djstripe/stripe_objects.py

```diff
@@ -393,10 +395,13 @@ def sync_from_stripe_data(cls, data):
         return instance
 
     def __str__(self):
+        """Return string repr of StripeObject."""
         return smart_text("<{list}>".format(list=", ".join(self.str_parts())))
 
 
 class StripeSource(PolymorphicModel, StripeObject):
+    """A model which connects a Customer object to a StripeObject."""
```

Comment:
Not quite. This is the base class for payment sources like StripeCard. The customer foreign key is inherited by subsclasses.